src/backend/.env.example

```
@@ -0,0 +1,17 @@
+
+# Optional: separate URL for tests (defaults to DATABASE_URL)
+TEST_DATABASE_URL="postgresql://welbeing:welbeing@localhost:5433/test_welbeing?schema=public"
+TEST_DATABASE_SCHEMA="jest_test"
+
+# Redis (optional; caching). App tolerates Redis being down.
+# If you use the compose redis service, it is not published to the host by default.
+REDIS_URL="redis://localhost:6379"
+
+PORT=3000
+NODE_ENV=development
+JWT_SECRET="change-me" # required in production; for local dev keep simple
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the NODE_ENV key

(UnorderedKey)